pyproject.toml

```diff
@@ -126,6 +126,7 @@ wemake-python-styleguide = "^0.19.2"
 flake8-pyproject = "^1.2.3"
 types-pyyaml = "^6.0.12"
 types-python-dateutil = "^2.9.0"
+icecream = "^2.1.3"
```

Comment:
Not used in source code

Reply:
Isn't it would be helpful for debuging in dev environment ?

Reply:
Ok